requirements.txt

```diff
@@ -1,6 +1,102 @@
+absl-py==0.11.0
```

Comment:
These requirements are quite constraining -- at a future point (TF2.4?) we should be able to relax == to >= and remove some of them

Reply:
Best practices here seem hard to find.